xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -80,8 +80,11 @@ CWinSystemWin32::CWinSystemWin32()
   CAESinkDirectSound::Register();
   CAESinkWASAPI::Register();
   CWin32PowerSyscall::Register();
-  m_irss.reset(new CIRServerSuite());
-  m_irss->Initialize();
+  if (CServiceBroker::GetSettings().GetBool("input.irserversuite"))
```

Comment:
why the setting is checked on startup only? If I disable IRSS support it will be disabled only on next start and vise versa. from usability viewpoint it looks very weird